src/include/drbd/resource_conf.rb

```diff
@@ -609,6 +609,16 @@ def autoGenerateNodeID(res_config)
       deep_copy(res_config)
     end
 
+    def ValidDeviceName
+      dev_name = Convert.to_string(UI.QueryWidget(Id(:n_devc), :Value))
```

Comment:
`Convert` is a deprecated module and it should not be used in new written code . I would simply use `#to_s`:

~~~
dev_name = UI.QueryWidget(Id(:n_devc), :Value).to_s
~~~

Reply:
Changed
Thanks!